Add IMU timeline documentation to Player docs

N-M-T opened the pull request.

papr commented on src/core/software/pupil-player.md:
```diff
@@ -236,7 +236,47 @@ Results are exported in the following files:
 
+#### IMU Timeline
+This plugin visualizes accelerometer and gyroscope data from Pupil Invisible recordings. It also fuses the 
+data using Madgwick's algorithm to yield drift-free orientation estimates in the pitch and roll axes.
```
```suggestion
data using [Madgwick's algorithm](https://github.com/xioTechnologies/Fusion) to yield drift-free orientation estimates in the pitch and roll axes.
```

papr commented on src/core/software/pupil-player.md:
```diff
@@ -236,7 +236,47 @@ Results are exported in the following files:
+Read more about [Pupil Invisible's coordinate systems here](/developer/invisible/#coordinate-systems).
+:::
+
+This Plugin is does not estimate orientation about the yaw axis (head rotation from left to right). This is 
```
```suggestion
This Plugin does not estimate orientation about the yaw axis (head rotation from left to right). This is 
```

papr reviewed (commented):
So from what I understand Sebastian Madgwick authored the original algorithm but xioTechnologies is now maintaining it. Maybe, we should give credit to both? In any case, I suggest to link to their official github repo, see below.

willpatera commented:
@papr @N-M-T are we awaiting further changes or shall I review?

N-M-T commented:
@papr I applied your suggestions, and updated and added a link to the Player Specific Plugin table. I don't think there is anything else to add, in which case it's ready for your review @willpatera
